python-threatexchange/threatexchange/extensions/vpdq/tests/test_vpdq_faiss.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+from cmath import sin
+import pytest
+from pathlib import Path
+import pickle
+
+try:
+    import vpdq as _
+
+    _DISABLED = False
+except (ImportError, ModuleNotFoundError) as e:
+    _DISABLED = True
+else:
+    import typing as t
+    from threatexchange.extensions.vpdq.vpdq_index import VPDQIndex
+    from threatexchange.extensions.vpdq.vpdq_util import (
+        prepare_vpdq_feature,
+        VPDQ_QUALITY_THRESHOLD,
+    )
+    from threatexchange.extensions.vpdq.video_vpdq import VideoVPDQSignal
+
+    from threatexchange.signal_type.index import (
+        VPDQIndexMatch,
+    )
+
+ROOTDIR = Path(__file__).parents[5]
+example_meta_data = {"name": "example_video"}
```

Comment:
Traditionally these are in ALL_CAPS

You can also type this as t.Mapping[str, str]  to enforce that it can't accidentally be changed.

You could also add `hash` here as well